turtle_sent_two.py

```diff
@@ -0,0 +1,229 @@
+import string
+
+
+class Message:
+    """
+    A Message class. Allows to store messages in an orderly fashion.
+
+    Parameters
+    ----------
+    m_id : int
+        Incremental id of the message sent.
+    _body : str
+        String of a body text of message.
+    sender : str
+        Name of a sender.
+
+    Attributes
+    ----------
+    message_id : int
+        Incremental id of the message sent.
+    message_body : str
+        String of a body text of message.
+    message_sender : str
+        Name of a sender.
+    message_read : bool
+        If the message already read or not.
+    """
+    def __init__(self, m_id, body, sender):
```

Comment:
Use type hints and return types -
https://www.geeksforgeeks.org/duck-typing-in-python/
https://stackoverflow.com/questions/125367/dynamic-type-languages-versus-static-type-languages
https://docs.python.org/3/library/typing.html
https://realpython.com/lessons/pros-and-cons-type-hints/